examples/resource_lacework_vulnerability_exception_container/fixable/main.tf

```diff
@@ -78,4 +84,8 @@ output "cves" {
 
 output "packages" {
   value = lacework_vulnerability_exception_container.example.vulnerability_criteria.0.package
+}
+
+output "fixable" {
+  value = lacework_vulnerability_exception_container.example.vulnerability_criteria.0.fixable
```

Comment:
The problem with your drift wasn't in the provider codebase, but instead here in your test.

You can use this code to store the output as `"null"` when the value of `fixable` is `null` 
```suggestion
  value = coalesce(lacework_vulnerability_exception_container.example.vulnerability_criteria.0.fixable, "null")
```